neurons/competition_runner.py

```diff
@@ -8,124 +7,47 @@
 from types import SimpleNamespace
 from datetime import datetime, timezone, timedelta
 import bittensor as bt
-from typing import List
-
-from competition_config import competitions as competitions_cfg
-
-import wandb
+from typing import List, Tuple, Dict
 
 # from cancer_ai.utils.config import config
 
 # TODO integrate with bt config
-path_config = SimpleNamespace(
-    **{"model_dir": "/tmp/models", "models_dataset_dir": "/tmp/datasets"}
+test_config = SimpleNamespace(
+    **{
+        "model_dir": "/tmp/models",
+        "dataset_dir": "/tmp/datasets",
+        "wandb_entity": "testnet",
+        "wandb_project_name": "melanoma-1",
+    }
 )
 
+main_competitions_cfg = json.load(open("neurons/competition_config.json", "r"))
 
-def calculate_next_evaluation_times(evaluation_times) -> List[datetime]:
-    """Calculate the next evaluation times for a given list of times in UTC."""
-    now_utc = datetime.now(timezone.utc)
-    next_times = []
 
-    for time_str in evaluation_times:
-        # Parse the evaluation time to a datetime object in UTC
-        evaluation_time_utc = datetime.strptime(time_str, "%H:%M").replace(
-            tzinfo=timezone.utc, year=now_utc.year, month=now_utc.month, day=now_utc.day
+def run_all_competitions(path_config: str, competitions_cfg: List[dict]) -> None:
```

Comment:
what is the use case of run_all_competitions?
I thought that we will only use the scheduler-like approach

Reply:
for debugging purposes, let it be